modules/internal/ChapelIO.chpl

```diff
@@ -141,34 +143,45 @@ function resolution error if the class NoRead is read.
 
   delete nr;
 
-.. _default-write-and-read-methods:
+.. _default-readThis-writeThis:
 
-Default write and read Methods
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Default writeThis and readThis Methods
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Default ``write`` methods are created for all types for which a user-defined
-``write`` method is not provided.  They have the following semantics:
+Default ``writeThis`` methods are created for all types for which a user-defined
+``writeThis`` or ``readWriteThis`` method is not provided.  They have the
+following semantics:
 
-* for an array argument: outputs the elements of the array in row-major order
-  where rows are separated by line-feeds and blank lines are used to separate
-  other dimensions.
-* for a `domain` argument: outputs the dimensions of the domain enclosed by
-  ``[`` and ``]``.
-* for a `range` argument: output the lower bound of the range, output ``..``,
-  then output the upper bound of the range.  If the stride of the range
-  is not ``1``, output the word ``by`` and then the stride of the range.
-* for a tuples, outputs the components of the tuple in order delimited by ``(``
-  and ``)``, and separated by commas.
 * for a class: outputs the values within the fields of the class prefixed by
   the name of the field and the character ``=``.  Each field is separated by a
   comma.  The output is delimited by ``{`` and ``}``.
 * for a record: outputs the values within the fields of the class prefixed by
   the name of the field and the character ``=``.  Each field is separated by a
   comma.  The output is delimited by ``(`` and ``)``.
 
-Default ``read`` methods are created for all types for which a user-defined
-``read`` method is not provided.  The default ``read`` methods are defined to
-read in the output of the default ``write`` method.
+Default ``readThis`` methods are created for all types for which a user-defined
+``readThis`` method is not provided.  The default ``readThis`` methods are
+defined to read in the output of the default ``writeThis`` method.
+
+Additionally, the Chapel implementation includes ``writeThis`` or methods for
```

Comment:
I think there's a missing word here, I doubt you meant to say "includes ``writeThis`` or methods".